catt/cli.py

```diff
@@ -89,9 +89,12 @@ def write_config(settings):
 
 @cli.command(short_help="Send a video to a Chromecast for playing.")
 @click.argument("video_url", callback=process_url)
+@click.option("-f", "--force-default", is_flag=True,
+              help="Force use of the default controller.")
```

Comment:
Can you explain a bit more what this does? End users probably won't know what a controller is.

Reply:
Fair point, I considered using "default media controller" instead, but figured that that wouldn't be any clearer. Does such elaboration belong in cli help texts?. I can't think of a way to explain this in a few words. Any ideas?

Reply:
Hmm, yeah, good point. I can't find a way to summarize it either. Maybe if it's in terms of "what will happen"? What's a reason that would make someone want to use the default controller?

Reply:
"default chromecast app", maybe?

Reply:
Yeah, that sounds better. Still, *why* would someone use that option? What's your use case for adding it? We can say something like "(use this if X doesn't work)".

Reply:
> What's a reason that would make someone want to use the default controller?

If a custom controller is broken.

Reply:
fixed